bigframes/operations/ai.py

```diff
@@ -0,0 +1,900 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import typing
+from typing import List, Optional
+import warnings
+
+import numpy as np
+
+from bigframes import dtypes, exceptions
+from bigframes.core import guid, log_adapter
+
+
+@log_adapter.class_logger
+class Ai:
```

Comment:
Python style says this should be AI not Ai.

https://peps.python.org/pep-0008/#descriptive-naming-styles

> When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.

Reply:
Done